pinecone/suite_runner_test.go

```diff
@@ -10,48 +10,44 @@ import (
 	"github.com/stretchr/testify/suite"
 )
 
+// This is the entry point for all integration tests
```

Comment:
Moved this up to the top of the file with a comment denoting it's ultimately the entrypoint.

Reply:
Nice, this is super helpful

Reply:
Should we update the readme with a lil outline of how the tests work, or is that overkill?

Reply:
Good point, I think I can add a section to the `CONTRIBUTING.md` file to explain some of the testing details.

Reply:
cool!